lib/minesweeper_2pl/mocks/mocked_cli.rb

```diff
@@ -0,0 +1,17 @@
+module Minesweeper_2pl
+  class MockedCli < CLI
+
+    def print(msg)
+    end
+
+    def ask_for_move
+    end
+
+    def get_player_input(game)
```

Comment:
This is a great start at rolling your own mocks! Nice job! What if you wanted to set up different player inputs in different test scenarios?